pkg/util/input.go

```diff
@@ -52,3 +53,21 @@ func AskForConfirmation(message string) bool {
 		" Do you want to continue [y/N]?")
 	return strings.ToUpper(userConfirmation) == "Y"
 }
+
+// Retrive password from OS native credential provider/keychain
+func GetPasswordKeyring(username string) string {
+	password, err := keyring.Get("minishift", username)
+	if err != nil {
+		return ""
```

Comment:
why `err` is not returned here and we are returning the empty string? if this is because in case user doesn't have keyring software install then it should not brag about it?

if we ignoring the error then why bother to evaluate because in case of error anyway password will be empty string right?

```
password, _ := keyring.Get("minishift", username)
return password
```